hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/admin/nssummary/NSSummaryCLIUtils.java

```diff
@@ -107,8 +108,9 @@ public static String makeHttpCall(StringBuffer url, String path,
     }
   }
 
-  public static HashMap<String, Object> getResponseMap(String response) {
-    return new Gson().fromJson(response, HashMap.class);
+  public static HashMap<String, Object> getResponseMap(String response)
```

Comment:
I don't think so, we need this wrapper method. Call JsonUtils method directly where needed.

Reply:
Yes thanks, I have completely removed the wrapper now we can directly call `JsonUtils.readTree(response);` to return a JsonNode and help us out.